src/api/kube.ts

```diff
@@ -1265,73 +1265,71 @@ export class KubeHelper {
     }
   }
 
-  async createCheClusterFromFile(filePath: string, flags: any, ctx: any, useDefaultCR: boolean): Promise<any> {
-    let yamlCr = this.safeLoadFromYamlFile(filePath)
-
+  async createCheCluster(CR: any, flags: any, ctx: any, useDefaultCR: boolean): Promise<any> {
```

Comment:
I am not sure that CR is a good name here. I think we should mention `che` or better `checluster`. What do you think about `CheClusterCR`?

Reply:
Done